lib/govuk_ab_testing/acceptance_tests/active_support.rb

```diff
@@ -0,0 +1,36 @@
+module GovukAbTesting
+  module AcceptanceTests
+    class ActiveSupport
+      attr_reader :request, :scope
+
+      def initialize(scope)
+        @request = scope.instance_variable_get(:@request)
```

Comment:
Check `@request` is defined/not `nil`? You do a check for `page` in `capybara.rb`